scripts/copy_bundles.py

```diff
@@ -26,6 +26,8 @@
 from azul.dss import (
     MiniDSS,
     shared_dss_credentials,
```

Comment:
https://github.com/DataBiosphere/azul/pull/1366/files#diff-61633c57a57b908ccba54e0971f8fd0dR26

Symbols in azul.dss should not duplicate the `dss` string. The way this is implemented here and now, `new_version` is a bit general and lacks context. But if references as `azul.dss.new_version` it becomes clear.